docs/markdown/Creating-releases.md

```diff
@@ -60,6 +60,16 @@ for example when done in CI that already does its own testing.
 So with `--no-tests` you can tell Meson "Do not build and test generated
 packages.".
 
+## Use `--allow-dirty` to override error when git repository contains uncommitted changes
+
+*Since 0.61.2* Instead of emitting a warning when a repository contains
```

Comment:
Obviously this number might need changing depending on the next release and when this PR is merged, but for now I put it there as a placeholder

Reply:
It will always be scheduled for the next major.minor release (a feature release, in this case probably 0.62.0), because we do not backport features into patch-level releases.